lib/shared/bucketing-assembly-script/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@devcycle/bucketing-assembly-script",
-  "version": "1.0.20-alpha.0",
+  "version": "1.0.20",
```

Comment:
do not change, this can stay as is with the alpha tag.. otherwise this will publish a new version of the bucketing-assembly-script with version `1.0.20`